pkg/api/v1alpha1/conversion.go

```diff
@@ -242,7 +242,7 @@ func enableProfilePluginsByType(profilePlugins api.Plugins, pluginInstance frame
 func checkBalance(profilePlugins api.Plugins, pluginInstance framework.Plugin, pluginConfig *api.PluginConfig) api.Plugins {
 	_, ok := pluginInstance.(framework.BalancePlugin)
 	if ok {
-		klog.V(3).Info("converting Balance plugin: %s", pluginInstance.Name())
+		klog.V(3).Info("converting Balance plugin: ", pluginInstance.Name())
```

Comment:
I think these were meant to be `Infof()` with the `%s`?

Reply:
commit now, please check